webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/policies/Rule.scala

```diff
@@ -55,13 +55,13 @@ final case class RuleRId(id: RuleId, revId: Option[RevId] = None)
  */
 final case class Rule(
     id              : RuleId
-  , revId           : Option[RevId]
```

Comment:
Revision are still option for Rules ? (maybe that intented has we focus on Directive here)

Reply:
yes, it will be in next iteration